test/functional/test_framework/test_framework.py

```diff
@@ -524,6 +524,9 @@ def set_dash_test_params(self, num_nodes, masterodes_count, extra_args=None, fas
         if fast_dip3_enforcement:
             for i in range(0, num_nodes):
                 self.extra_args[i].append("-dip3params=30:50")
+        if whitelist_peers:
+            for i in range(0, num_nodes):
+                self.extra_args[i].append("-whitelist=127.0.0.1")
```

Comment:
The reason was speeding up propagation between non-masternodes and masternodes. But this was before I did all the other fixes, so this might have gotten unnecessary. I'll test without this...

Reply:
Turned out that tests are green even without this change. Force pushed without this commit